tests/unit/core/highlight-vars-spec.js

```diff
@@ -1,13 +1,14 @@
 "use strict";
 
-import { flushIframes, makeRSDoc, makeStandardOps } from "../SpecHelper.js";
-
-// TODO: add tests:
-// - right colors are applied
+import { flushIframes, makePluginDoc } from "../SpecHelper.js";
 
 describe("Core - highlightVars", () => {
   afterAll(flushIframes);
-  const testBody = `
+
+  const plugins = ["/src/core/highlight-vars.js"];
```

Comment:
Using absolute paths from /src as it gets a little better editor support (not including renaming)